ports/openldap/portfile.cmake

```
@@ -0,0 +1,59 @@
+vcpkg_download_distfile(ARCHIVE
+    URLS "https://www.openldap.org/software/download/OpenLDAP/openldap-release/openldap-2.5.13.tgz"
+    FILENAME "openldap-2.5.13.tgz"
+    SHA512 30fdc884b513c53169910eec377c2ad05013b9f06bab3123d50d028108b24548791f7f47f18bcb3a2b4868edeab02c10d81ffa320c02d7b562f2e8f2fa25d6c9
+)
+
+execute_process(COMMAND bash "-c" "awk -F= '/^NAME/{print $2}' \"/etc/os-release\""
+    OUTPUT_VARIABLE OS_RELEASE
+    ERROR_VARIABLE OS_RELEASE_ERROR
+    OUTPUT_STRIP_TRAILING_WHITESPACE
+)
+message("OS_RELEASE (OS_RELEASE_ERROR): ${OS_RELEASE} (${OS_RELEASE_ERROR})")
+
+if(OS_RELEASE STREQUAL "\"CentOS Linux\"")
+    vcpkg_extract_source_archive_ex(
+        OUT_SOURCE_PATH SOURCE_PATH
+        ARCHIVE ${ARCHIVE}
+        PATCHES
+        openssl.patch
+        m4.patch
+    )
+else()
+    vcpkg_extract_source_archive_ex(
+        OUT_SOURCE_PATH SOURCE_PATH
+        ARCHIVE ${ARCHIVE}
+        PATCHES
+        openssl.patch
+    )
+endif()
+
+if(VCPKG_LIBRARY_LINKAGE STREQUAL dynamic)
+    set(BUILD_OPTS --enable-shared=yes --enable-static=no)
+else()
+    set(BUILD_OPTS --enable-shared=no --enable-static=yes)
+endif()
+
+vcpkg_configure_make(
+    SOURCE_PATH "${SOURCE_PATH}"
+    AUTOCONFIG
+
+    OPTIONS
+    ${BUILD_OPTS}
+    --disable-slapd
+    --with-tls=openssl
+    --without-cyrus-sasl
+    "LIBS=-ldl"
+)
+
+vcpkg_build_make(BUILD_TARGET depend)
+vcpkg_build_make()
+vcpkg_install_make()

```

```suggestion
vcpkg_install_make()
```
`vcpkg_install_make()` will automatically call `vcpkg_build_make()`, if `BUILD_TARGET` is not passed, the default is all.

---

And `vcpkg_install_make` will overwrite the build logs from `vcpkg_build_make`.

---

vcpkg_build_make(BUILD_TARGET depend) will still needed to be called prior to vcpkg_install_make(), but I will remove the extra vcpkg_build_make()

---

I have removed the second call to vcpk_build_make and added the LOGFILE setting